### ISSUE TITLE
OwnedObject lacks a create method

### ISSUE THREAD

**Issue by tcmitchell** [llotneb/SBOL#31]
A common pattern of use is `doc.xxx.create('foo')` to create new objects of a given type. Here are two examples from the SBOL tutorial:

```
my_device = doc.componentDefinitions.create('my_device')
```

```
design = doc.designs.create('my_design')
```

`doc.designs` and `doc.componentDefinitions` are both of type `OwnedObject`. 

* Add a class argument to the OwnedObject constructor so that the future `create()` method knows what type of object to create. 
  * For backward compatibility this should be the third argument
* Add a create method that takes a string argument
* Change all the uses of OwnedObject to include a class argument
* Add class `OwnedPythonObject` for backward compatibility (see [pySBOLx.py](https://github.com/nroehner/pySBOLx/blob/master/pySBOLx/pySBOLx.py#L82))

**Comment by tcmitchell** [comment]
Note: class `OwnedObject` is not exported in pySBOL, so it is not part of the public API. `OwnedPythonObject` _is_ part of the public API, so we should make a reasonable effort to maintain that for backward compatibility.

**Comment by tcmitchell** [comment]
Closed by #33
